main.go

```diff
@@ -1052,11 +1053,27 @@ func main() {
 
 	mainLog.Info("Stop signal received.")
 
+	//if using async session writes stop workers
+	if config.Global().UseAsyncSessionWrite {
```

Comment:
there will be also two pools of workers (`spec.SessionManager` and `spec.OrgSessionManager`) running per each API spec started here https://github.com/TykTechnologies/tyk/blob/v2.6.2/api_loader.go#L188 so we need to stop them as well before exit. Maybe just go through all loaded specs and stop pools